pkg/scheduler/algorithm/predicates/metadata.go

```diff
@@ -30,7 +30,8 @@ import (
 )
 
 type PredicateMetadataFactory struct {
-	podLister algorithm.PodLister
+	podLister                 algorithm.PodLister
```

Comment:
There should be only one space between `podLister` and `algorithm.PodLister`, or it will cause a golint error.

Reply:
This is the effect of the gofmt formatting and the alignment below.